app/models/qdm/tacoma/valueset.rb

```diff
@@ -0,0 +1,68 @@
+module QDM
+  # ValueSet represents a collection of Concepts, used by the Measures to specify a set of codes for a particular topic
+  class ValueSet
+    include Mongoid::Document
+    field :oid, type: String
+    field :display_name, type: String
+    field :version, type: String
+    field :user_id, type: String # Eventually we need to delete this from bundles when exporting
+    field :categories, type: Hash
+
+    index(oid: 1)
+    embeds_many :concepts
+    index 'concepts.code' => 1
+    index 'concepts.code_system' => 1
+    index 'concepts.code_system_name' => 1
+    index 'bundle_id' => 1
```

Comment:
This needs to be considered further; may not be needed in Bonnie, needed in Cypress/exported bundles